tests/js/js2py/boolean.simple

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file    js2py/boolean.simple - Asserts that sending primitive and boxed
+ *          booleans to Python and getting them back into JS as primitive
+ *          booleans retains the correponsing logical values.
+ *
+ * @author  Bryan Hoang <bryan@ditributive.network>
```

Comment:
your e-mail address is bryan@distributive.network 🥇 🤣

Reply:
Whoops! I've added the missing 's'.